src/namer.h

```diff
@@ -142,6 +149,11 @@ class Namer {
     return result;
   }
 
+  std::string NamespacedType(const std::vector<std::string> &ns,
+                             const std::string &s) const {
+    return Namespace(ns) + config_.namespace_seperator + Type(s);
```

Comment:
Some languages have a prefix ("::" or "global::") to represent the global namespace. Do you think we should add that configuration here as well?

Reply:
I think we should centralize the configuration if there's more than one or two languages. We can add that as/when we need it.